base/setup/lib/utils/arcname.c

```diff
@@ -119,7 +119,7 @@ ArcGetNextTokenA(
 {
     NTSTATUS Status;
     PCSTR p = ArcPath;
-    ULONG SpecifierLength;
+    SIZE_T SpecifierLength;
```

Comment:
That variable also needs to be converted to SIZE_T in the ArcGetNextTokenU function below.

Reply:
done